apps/fz_http/test/fz_http/repo/notifier_test.exs

```diff
@@ -0,0 +1,76 @@
+defmodule FzHttp.Repo.NotifierTest do
+  use ExUnit.Case, async: false
+
+  import FzHttp.TestHelpers
+
+  alias Ecto.Adapters.SQL.Sandbox
+  alias FzHttp.Events
+  alias FzHttp.Repo
+
+  setup do
+    start_supervised!({Postgrex.Notifications, [name: Repo.Notifications] ++ Repo.config()})
+    start_supervised!(Repo.Notifier)
+    :ok = Sandbox.checkout(Repo)
+  end
+
+  test "adds and removes user from wall state on db notifications" do
+    Sandbox.unboxed_run(Repo, fn ->
```

Comment:
Was `unboxed_run/2` needed here to execute the triggers? Thinking in general this a really bad idea because it can affect other tests. Curious if there's another way

Reply:
Unfortunately, yes. The sandbox is one big transaction and since these triggers fire after a transaction commits, we will never see them in normal the sandboxed tests.

Reply:
Kk